src/LaravelExactOnline.php

```diff
@@ -458,7 +458,7 @@ public static function loadConfig()
             );
         }
 
-        return (object) json_decode($config, false);
+        return (object)json_decode($config, false);
```

Comment:
Could you please revert this change to keep the current code style?

Reply:
sure <3